packages/suite/src/reducers/wallet/coinjoinReducer.ts

```diff
@@ -81,12 +98,19 @@ export const coinjoinReducer = (
                 // restore accounts with coinjoin accountType
                 action.payload.accounts.forEach(account => {
                     if (account.accountType === 'coinjoin') {
-                        createAccount(draft, account);
+                        draft.accounts.push({
+                            key: account.key,
+                            targetAnonymity: 0,
```

Comment:
I think that causes `targetAnonymity` to reset when a device is remembered and page is reloaded.

Reply:
following 4 lines above:
```
// Temporary code
                // coinjoin reducer is not stored in DB yet
                // coinjoin reducer is not stored in DB yet
                // restore accounts with coinjoin accountType
                // restore accounts with coinjoin accountType
```

it will be addressed in separate PR